src/daft-local-execution/src/pipeline.rs

```diff
@@ -349,30 +349,10 @@ pub fn physical_plan_to_pipeline(
                 _ => true,
             };
 
-            // TODO(desmond): We might potentially want to flip the probe table side for
-            // left/right outer joins if one side is significantly larger. Needs to be tuned.
-            //
-            // In greater detail, consider a right outer join where the left side is several orders
-            // of magnitude larger than the right. An extreme example might have 1B rows on the left,
-            // and 10 rows on the right.
-            //
-            // Typically we would build the probe table on the left, then stream rows from the right
-            // to match against the probe table. But in this case we would have a giant intermediate
-            // probe table.
-            //
-            // An alternative 2-pass algorithm would be to:
-            // 1. Build the probe table on the right, but add a second data structure to keep track of
-            //    which rows on the right have been matched.
-            // 2. Stream rows on the left until all rows have been seen.
-            // 3. Finally, emit all unmatched rows from the right.
             let build_on_left = match join_type {
-                JoinType::Inner => build_on_left,
```

Comment:
i think it'll be a little be safer to exhaust all the cases 
```
JoinType::Inner | JoinType::Outer | ... => build_on_left
```

Reply:
Makes sense, implemented.